packages/notebook-extension/style/base.css

```
@@ -0,0 +1,11 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+.jp-cell-filters-types,
+.jp-cell-filters-tags {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
```

There is no value for user if double click selects the word "code"/"markdown"/"raw" in the label:

```suggestion
}

.jp-cell-filters-types {
  user-select: none;
}
```